### PR TITLE
Update evaluation

### PR DESCRIPTION
**Pull request by yeganehkordi**
Main changes in evaluation.py:
- Completing the Evaluation class
- Covering new input types
- Randomly testing n instances from each dataset
- Calculating the task id based on the table on the main page of Turkle

Other changes:
- Dropping extra scripts
- Changing the upload script to read input files

### CODE REVIEW

**Comment by danyaljj** [comment]
Closes #16

File: src/evaluation.py [hunk 1]
```diff
@@ -13,26 +14,28 @@
...
-    def calculate_rouge(self, n, input_name, baseline_answer):
-        # TBD
-        df = pd.read_csv('batch.csv')
-        start_index = (n * 3) - 1
-        answers = []
-        for i in range(3):
-            answer = self.df.iloc[start_index + i][f'Answer.{input_name}']
-            answers.append(answer)
-
+    def calculate_rouge(self, task, index, input_name, baseline_answer):
```
Comment (danyaljj) [inline src/evaluation.py:25]: 💯

File: src/evaluation.py [hunk 2]
```diff
@@ -64,8 +67,73 @@ def enter_input(input_type, input_value, input_name, driver):
...
+            elif input_type == 'time':
+                time_picker = driver.find_element(By.NAME, input_name) 
+                time_picker.click() 
+                time_picker.send_keys(input_value) 
+            elif input_type == 'url': 
+                url_input = driver.find_element(By.NAME, input_name) 
+                url_input.click() 
+                url_input.send_keys(input_value) 
+        except:
+            print(f"We have a problem with '{input_name}'")
```
Comment (danyaljj) [inline src/evaluation.py:136]: awesome!

File: src/evaluation.py [hunk 3]
```diff
@@ -139,49 +211,53 @@ def solve_task(task):
 
 
 def find_frame_url(project_name, driver):
```
Comment (danyaljj) [inline src/evaluation.py]: Should this function be something else? I am not sure if it is returning a URL.

**Comment by danyaljj** [comment]
One minor suggestion regarding a function name. Otherwise good to go! 
Merge when you're happy.

File: src/evaluation.py (hunk 3 above)
Comment (yeganehkordi) [reply src/evaluation.py]: Yeah, I need to change its name

**Pull request merged**